File: spring-cloud-config/config-jdbc-client/src/main/java/com/livk/client/ConfigJdbcClient.java
package com.livk.client;

import com.livk.commons.spring.LivkSpring;
import org.springframework.beans.factory.annotation.Value;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

/**
 * <p>
 * ConfigClient
 * </p>
 *
 * @author livk
 */
@RestController
@SpringBootApplication
public class ConfigJdbcClient {

    @Value("${foo}")
    String foo;

    public static void main(String[] args) {
        LivkSpring.run(ConfigJdbcClient.class, args);
    }

    @RequestMapping(value = "/foo")
    public String hi() {
        return foo;
    }

}

File: spring-cloud-config/config-jdbc-client/src/main/kotlin/com/livk/client/ConfigJdbcClient.kt
package com.livk.client

import org.springframework.beans.factory.annotation.Value
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController

/**
 * @author livk
 */
@RestController
@SpringBootApplication
open class ConfigJdbcClient {
    @Value("\${foo}")
    lateinit var foo: String

    @RequestMapping(value = ["/foo"])
    fun hi(): String = foo
}

fun main(args: Array<String>) {
    runApplication<ConfigJdbcClient>(*args)
}
